httpd/httpd.conf

```diff
@@ -68,6 +68,9 @@ DocumentRoot "/app"
 <IfDefine HTTPD_IN_SUBDIR>
 Alias "/bmo" "/app"
 </IfDefine>
+<IfDefine HTTPS>
+    SetEnv HTTPS ON
+</IfDefine>
```

Comment:
when httpd sets HTTPS, it uses a lowercase "on" – this should be lowercase too to match, and code that checks $ENV{HTTPS} needs to be updated to use "on" instead of "ON".